search-sentiments-map/test/trends-test.js

```diff
@@ -0,0 +1,66 @@
+const mocha = require('mocha');
+const chai = require('chai');
+const assert = chai.assert;
+const sinon = require('sinon');
+const trends = require('./../routes/trends').trends;
+
+describe('Trends', function() {
+  describe('getGlobalTrends', function() {
+    let mockResult;
+    let mockEmpty;
+    beforeEach(() => {
+      mockResult = [
+        {trendTopic: 'Donald Trump', count: 3},
+        {trendTopic: 'Coronavirus', count: 2},
+        {trendTopic: 'Dogs', count: 1}
+      ];
+      mockEmpty = [];
```

Comment:
Why do you put those two in beforeEach()? I think you can put them in the two "it"s separately cuz they are only used by one

Reply:
Its convention to declare variables outside the loop, so I'd say let it be there.

Reply:
Yeah leave it here.